protocol.md

```diff
@@ -1,55 +1,248 @@
+| N bytes | Ticker name of this the coin configuration. The ticker has to be the same as the FROM ticker                            |
+| 1 byte  | Application length                                                                                                      |
+| N bytes | Name of the application that can handle this currency                                                                   |
+| 1 byte  | Sub coin configuration length                                                                                           |
+| N bytes | Sub coin configuration, used for tokens to specify an app the subticker and the decimals                                |
+| N bytes | Signature of the coin configuration by the Ledger key, curve secp256k1 hashfunc sha256                                  |
+| 1 byte  | Packed derivation path length                                                                                           |
+| N bytes | Packed derivation path used for the FROM coin                                                                           |
+
+#### CHECK_ASSET_IN
+
+This command is used only in the SELL_NEW and FUND_NEW TYPES. \
+The data content is the same as CHECK_ASSET_IN_LEGACY, only the command id is different.
+
+#### CHECK_PAYOUT_ADDRESS
+
+This command is used only in the SWAP_LEGACY and SWAP_NEW TYPES.
+
+| Bytes   | Description                                                                                                           |
+| ------- | --------------------------------------------------------------------------------------------------------------------- |
+| 1 byte  | Coin configuration length of the TO coin. The coin configuration is made of the ticker, appname, and subconfiguration |
+| 1 byte  | Ticker length                                                                                                         |
+| N bytes | Ticker name of this the coin configuration. The ticker has to be the same as the TO ticker                            |
+| 1 byte  | Application length                                                                                                    |
+| N bytes | Name of the application that can handle this currency                                                                 |
+| 1 byte  | Sub coin configuration length                                                                                         |
+| N bytes | Sub coin configuration, used for tokens to specify an app the subticker and the decimals                              |
+| N bytes | Signature of the coin configuration by the Ledger key, curve secp256k1 hashfunc sha256                                |
+| 1 byte  | Packed derivation path length                                                                                         |
+| N bytes | Packed derivation path used for the TO coin                                                                           |
```

Comment:
As before, as `N` changes, it should be documented with other names.

Reply:
Done